### PR TITLE
Fix time comparison when determining if file should roll based on time

### PR DESCRIPTION
**Pull request by bbalser**

### CODE REVIEW

**Review by madninja — APPROVED** [review]

**Review by dpezely — APPROVED** [review]

**Review by jeffgrunewald — APPROVED** [review]

File: file_store/src/file_sink.rs [hunk 1]
```diff
@@ -294,7 +294,7 @@ impl FileSink {
 
     pub async fn maybe_roll(&mut self) -> Result {
         if let Some(active_sink) = self.active_sink.as_mut() {
-            if active_sink.time + self.roll_time > Utc::now() {
+            if active_sink.time + self.roll_time < Utc::now() {
```
Comment (jeffgrunewald) [inline file_store/src/file_sink.rs:297]: i assume these timers aren't so precise that we'd need this check to be `<=`? i.e. it's not likely to hit directly on `active_sink.time + self.roll_time == Utc::now()` and keep cycling?

Comment (vihu) [inline file_store/src/file_sink.rs:297]:
```suggestion
            if (active_sink.time + self.roll_time) < Utc::now() {
```

**Review by jeffgrunewald — APPROVED** [review]

File: file_store/src/file_sink.rs [hunk 2]
```diff
@@ -16,6 +16,12 @@ use tokio::{
 use tokio_util::codec::{length_delimited::LengthDelimitedCodec, FramedWrite};
 
 pub const DEFAULT_SINK_ROLL_MINS: i64 = 3;
+
+#[cfg(not(test))]
+pub const DEFAULT_SINK_CHECK_MILLIS: i64 = 60_000;
```
Comment (madninja) [inline file_store/src/file_sink.rs:21]: Rename without DEFAULT prefix since this is not looked up anywhere

Comment (bbalser) [reply file_store/src/file_sink.rs]: Fixed 536f3c1

**Pull request merged**